src/collections/db.ts

```diff
@@ -64,7 +64,7 @@ export class Db {
    * @param options
    * @returns Promise
    */
-  async createCollection(collectionName: string, options?: CreateCollectionOptions) {
+  async createCollection(collectionName: string, options?: Record<string, any>) {
```

Comment:
I'd prefer to keep `CreateCollectionOptions` here just to keep the TypeScript types consistent